contents/docs/posthog-ai/pricing-faq.mdx

```diff
@@ -0,0 +1,62 @@
+# PostHog AI credits
+
+PostHog uses **AI credits** to charge for all of its AI features.  
+Each time you interact with an AI-powered feature — like asking PostHog to write SQL, summarize data, or analyze sessions — a small number of credits are used based on the complexity of your request.
+
+This means you only pay for what you actually use.  
+There’s no fixed subscription — and every organization has a free tier worth $25 (2,500 credits) so you can explore all PostHog’s AI features before paying anything.
+
+### **When AI credits are used**
+
+AI credits are consumed whenever PostHog performs intelligent work for you.  
+You’ll spot these features by the ✨ icon or when using the in-app chat.
+
+Examples include:
+- Searching for session recordings in natural language  
+- Summarizing experiment results  
+- Writing or debugging SQL with AI  
+
+### **What doesn’t use credits**
+
+Getting help on how to use PostHog never costs credits.  
+All AI features still in beta are also free to use — as of **Oct 31**, that includes **session summaries** and **deep research**.
+
+### **How credit usage works**
+
+AI credits are based on **token costs**, which reflect the effort required to complete your request.
+
+- Simple queries (like “What were my daily active users in October?”) use very few tokens, and therefore very few credits.  
+- More complex tasks (like analyzing hundreds of session recordings or rewriting a SQL query multiple times) use more tokens and cost a bit more.  
+
+While exact usage varies, credit consumption usually scales with value — more advanced tasks cost more but deliver deeper insights and time savings.
+
+You’ll always see **real-time cost information** while using AI features.
+
+To keep it simple, token costs are converted into **AI credits**, billed at **$0.01 per credit** — so **1,000 credits = $10**.
+
+| Example interaction | Estimated credit cost |
+| --- | --- |
+| “What are my daily active users in October?” | coming soon |
+| “Show me session recordings for this organization_id” | coming soon |
+| “Can you help me write a SQL query that…” | coming soon |
+
+---
+
+### **Track your AI credit usage**
+
+Stay on top of your AI credit spend in real time:
+
+1. Type `/usage` in chat to see the current cost of your conversation  
+2. Check the **Billing & usage** page to view your total monthly consumption  
+3. (Coming soon) A dashboard showing which AI features use the most credits  
+
+> 💡 Credit data may take up to 24 hours to appear on the Billing & usage page.
+
+---
+
+### **Control your AI credit spend**
+
+You’re always in control of your AI credit usage:
+- **Real-time tracking** — see costs as you go  
+- **Billing limits** — set hard caps to prevent overspending  
+- **Usage alerts** — get notified when you hit key thresholds  
```

Comment:
Are these the standard billing alerts when you hit 80% and 100% of your free allowance (if on free plan) or billing limit (if on paid plan)? Or something custom for AI?

Reply:
Yes, standard. We don't have anything custom planned for now